src/Opdex.Platform.WebApi/Validation/VaultGovernances/VaultGovernanceCertificateFilterParametersValidator.cs

```diff
@@ -9,8 +9,9 @@ public class VaultGovernanceCertificateFilterParametersValidator : AbstractCurso
 {
     public VaultGovernanceCertificateFilterParametersValidator()
     {
-        RuleFor(request => request.Holder).MustBeNetworkAddressOrEmpty();
-        RuleFor(request => request.Status).MustBeValidEnumValueOrDefault();
-        RuleFor(filter => filter.Limit).LessThanOrEqualTo(Cursor.DefaultMaxLimit);
+        RuleFor(request => request.Holder).MustBeNetworkAddressOrEmpty().WithMessage("Holder must be valid address.");
+        RuleFor(request => request.Status).MustBeValidEnumValueOrDefault().WithMessage("Status must be valid or the enumeration values.");
```

Comment:
Maybe "Status is invalid, must be a valid enumeration value." or something simple like "Unknown status."